services/contract_update.proto

```diff
@@ -109,4 +109,27 @@ message ContractUpdateTransactionBody {
      * account. Otherwise it updates the contract's auto-renew account to the referenced account.
      */
     AccountID auto_renew_account_id = 12;
+
+    /**
+     * ID of the new account or node to which this contract is staking. It can never happen that staked_account_id
+     * is non-null and staked_node_id is non-negative at the same time.
+     */
+    oneof staked_id {
+
+        /**
+         * ID of the new account to which this contract is staking. If not staked to an account, the value is null.
+         * The default value is null.
+         */
+        AccountID staked_account_id = 13;
+
+        /**
+         * ID of the new node this contract is staked to. If not staked to a node, the value is -1. The default value is -1.
+         */
+        google.protobuf.Int64Value staked_node_id = 14;
+    }
+
+    /**
+     * If true, the contract declines receiving a staking reward. The default value is false.
```

Comment:
Since it's BoolValue, the default is the field is not set instead of `false`.

```suggestion
     * If true, the contract declines receiving a staking reward.
```

Reply:
Left the default value in place and fixed

Reply:
But the default value is unset not false. If you don't supply an explicit value the field will not be updated.